src/WebJobs.Script.WebHost/Models/RunFromPackageContext.cs

```diff
@@ -34,6 +34,11 @@ public bool IsRunFromPackage(ScriptApplicationHostOptions options, ILogger logge
             return (IsScmRunFromPackage() && ScmRunFromPackageBlobExists(options, logger)) || (!IsScmRunFromPackage() && !string.IsNullOrEmpty(Url) && Url != "1");
         }
 
+        public bool IsRunFromLocalPackage()
+        {
+            return string.Equals(Url, "1", StringComparison.OrdinalIgnoreCase);
```

Comment:
Shouldn't this method also check if `WEB_RUN_FROM_PACKAGE` is set?

Reply:
It is now checking for `WEBSITE_RUN_FROM_PACKAGE`